[docs] add Python binary/packages examples (#395)
kestra-io/kestra.io: content/docs/05.developer-guide/03.scripts.md

      key: gpu
```    

> Also note that `PROCESS` can be used with Kestra running on Docker. You might need to bring your own binary into the Dockerfile, as [explained in this part](../09.administrator-guide/02.deployment/01.docker.md).

Reviewer: ```suggestion
Also, note that `PROCESS` can be used with Kestra running in a Docker container. You can add your custom binaries into the Dockerfile, as [explained in this part](../09.administrator-guide/02.deployment/01.docker.md).
```
